ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -114,12 +113,9 @@ extension UIColor {
         var redValue: CGFloat = 0.0
         var greenValue: CGFloat = 0.0
         var blueValue: CGFloat = 0.0
-        if self.getRed(&redValue, green: &greenValue, blue: &blueValue, alpha: nil) {
-            // Ensure that all channels fall within 0x0 and 0xFF
-            let red = min(UInt32(redValue * 255.0), 0xFF)
-            let green = min(UInt32(greenValue * 255.0), 0xFF)
-            let blue = min(UInt32(blueValue * 255.0), 0xFF)
-            let colorValue: ColorValue = (red << 16) + (green << 8) + blue
+        var alphaValue: CGFloat = 0.0
```

Comment:
default ain't1 ?

Reply:
Shouldn't matter. I only need to initialize them to _something_ so I can pass the reference into `UIColor.getRed(_:green:blue:alpha:)`, and if that method fails then I don't use the values at all.

Maybe it's my C days showing, but 0 just seems more natural to initialize with than 1 🤔

Reply:
i don't know. i feel like yeah initilizing by false is natural in c so 0. but then., if alpha is 0 then... nothing is showing in the ui world... soo....

Reply:
Ehh true. Fiine I fix it.